saleor/graphql/discount/types.py

```diff
@@ -64,6 +66,9 @@ class Voucher(CountableDjangoObjectType):
     countries = graphene.List(
         CountryDisplay,
         description='List of countries available for the shipping voucher.')
+    translation = graphene.Field(
+        VoucherTranslation, language_code=graphene.String(required=True),
```

Comment:
We should add the description for the `language_code` argument e.g. "A language code to return the translation for".

Reply:
We should add it similarly to other translation fields in the API.